appsTransport.go

```diff
@@ -63,7 +63,7 @@ func (t *AppsTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	claims := &jwt.StandardClaims{
 		IssuedAt:  time.Now().Unix(),
 		ExpiresAt: time.Now().Add(time.Minute).Unix(),
-		Issuer:    strconv.Itoa(t.appID),
+		Issuer:    strconv.FormatInt(t.appID, 10),
```

Comment:
I'm OK with this, but what was the reason for this change?

Reply:
Itoa is only for int. FormatInt is the int64 equivalent. See https://godoc.org/strconv#Itoa

Reply:
oh of course, I only looked at the return parameters, not the arguments.